SPECS/k3s/k3s.spec

```diff
@@ -23,9 +23,9 @@ Source0: https://github.com/k3s-io/%{name}/archive/refs/tags/v%{version}+k3s1.ta
 # 3. cd %%{name}-%%{version}-k3s1
 # 4. go mod vendor
 # 5. pushd vendor
-# 6. git clone https://github.com/containerd/containerd.git -b release/1.6
+# 6. git clone https://github.com/k3s.io/containerd.git -b 1.5.13-k3s1
 # 7. git clone https://github.com/rancher/plugins.git -b k3s-v1.1.1
-# 8. git clone https://github.com/opencontainers/runc.git -b release-1.1
+# 8. git clone https://github.com/opencontainers/runc.git -b v1.1.3
 # 9. popd
 # 10. tar -cf %%{name}-%%{version}-vendor.tar.gz vendor
 Source1: %{name}-%{version}-vendor.tar.gz
```

Comment:
When I try to build it locally, I get a build failure when applying patch1: `install.patch`:
```bash
3 out of 3 hunks FAILED -- saving rejects to file package/rpm/install.sh.re
```

I would request you to please:
-  if `patch1` is actually valid/applicable with the bump from `1.23.6 -> 1.24.3` (we might need to refresh it; if it is applicable)
- Also, please run a buddy build with the spec :) [This makes life of reviewers easy & we also ensure we are not breaking the build]

Reply:
I have resolved point 1 and 2. Can you provide some more context with regards to point 3. Is there any documentation I can follow to run this build.

Reply:
Ohh sure! please feel free to reach-out offline; I will be happy to share pointers :)

Reply:
Update: we had an offline sync-up and shared a few pointers and best practices to follow :)